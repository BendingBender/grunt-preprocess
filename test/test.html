<!DOCTYPE html>
<html>
<head>
    <!-- exclude NODE_ENV='production' -->
    <title>dev title</title>
    <!-- endexclude -->

  <!-- exclude NODE_ENV='development' -->
  <title>prod title</title>
  <!-- endexclude -->

</head>
<body>
  <h1><!-- insert HOME --></h1>
</body>
</html>
