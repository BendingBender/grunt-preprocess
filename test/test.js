/*global define*/

define([], function () {
  "use strict";

  //exclude env='production'
  superExpensiveFunction()
  //endexclude

});
