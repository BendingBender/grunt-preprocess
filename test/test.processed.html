<!DOCTYPE html>
<html>
<head>
    
    <title>dev title</title>
    
    
</head>
<body>

</body>
</html>
