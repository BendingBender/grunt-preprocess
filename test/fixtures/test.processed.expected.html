<!DOCTYPE html>
<html>
<head>
    
    <title>dev title</title>
    

  

</head>
<body>
  <h1>/Users/joverson</h1>
  <h2>!foobar!
</h2>
</body>
</html>
